### DISCUSSION TITLE
Cannot install for mantine v7.3+

### DISCUSSION THREAD

**Discussion by viettbq2002** [KevinVandy/mantine-react-table#240] [Q&A]
I was reading the installation documents that said `mantine-react-table V1\2 requires Mantine V7.3+ packages as dependencies in your project.`
but my installation for mantine v7.3.2 was end up with this error

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: smar-lims-frontend@0.0.0
npm ERR! Found: @mantine/core@7.3.2
npm ERR! node_modules/@mantine/core
npm ERR!   @mantine/core@"^7.3.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @mantine/core@"^6.0" from mantine-react-table@1.3.4
npm ERR! node_modules/mantine-react-table
npm ERR!   mantine-react-table@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by alessandrojcm** [comment]
What version are you trying to install? MRT V1 only works with Mantine V6 and MRT V2 only works with Mantine V7

**Reply by viettbq2002** [reply]
I install MRT v2 base on this installation guide (https://v2.mantine-react-table.com/docs/getting-started/install)

**Comment by KevinVandy** [comment]
Looks like you installed MRT version 1 still. If using Mantine v7, you have to install the MRT v2 alpha

**Reply by viettbq2002** [reply]
I install MRT v2 base on the installation guide in documents https://v2.mantine-react-table.com/docs/getting-started/install
I already have mantine 7.3.2 in my project then `npm i mantine-react-table` occur this error
try to double-check by running `npm i mantine-react-table @mantine/core @mantine/hooks @mantine/dates @tabler/icons-react dayjs clsx` but seem also not working 
I also remove mantine  7 in my project and install using the above command, it installs success but mantine version 6 mantine table version 1.3.4

**Reply by KevinVandy** [reply]
The warning is literally telling you that you accidentally installed MRT v1.3.4. Just upgrade to the latest alpha version
